CMakeLists.txt

```diff
@@ -82,7 +82,7 @@ add_compile_options(${NMODL_COMPILER_WARNING_SUPPRESSIONS})
 # =============================================================================
 project(
   NMODL
-  VERSION ${NMODL_GIT_LAST_TAG}
+  VERSION "1.0"
```

Comment:
For some reason with only this branch I get the following:
```
CMake Error at CMakeLists.txt:83 (project):
  VERSION "v1.0-alpha-llvm" format invalid.
```
`v1.0-alpha-llvm` is the last `NMODL` tag